src/progressBar.test.js

```diff
@@ -0,0 +1,13 @@
+  // FOR SCREEN READERS
+// TEST CASES for updating aria-live region. Note: the values are made up. 
+  if (tagsCurrentCount === 6) {
+      let ariaMessage = `Your progress: ${tagsCurrentCount} tags added. This is 5%. Great job.`;
+      let progressLive = document.getElementById('progress-live');
+      progressLive.textContent = ariaMessage;
+  } 
+  if (tagsCurrentCount === 12) {
+      let ariaMessage = `Your progress: ${tagsCurrentCount} tags added. This is 10%. Great job.`;
+      let progressLive = document.getElementById('progress-live');
+      progressLive.textContent = ariaMessage;
+    }
+// END OF TEST CASES
```

Comment:
This was a great idea! We do need these tests since getting to 50% takes too long when we want to test that everything works great with screen readers, but it's good to have them out of the way.